src/components/atoms/VisusalisationsSelector.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { SelectField } from 'evergreen-ui';
+
+type Props = {
+    label: string;
+    options: Record<string, string>;
+    onChange: React.ChangeEventHandler<HTMLSelectElement>;
+};
+
+const VisualisationsSelector: React.FC<Props> = (props) => (
+    <SelectField label={props.label} defaultValue={Object.values(props.options)[0]} onChange={props.onChange}>
+        {Object.entries(props.options).map(([label, value]) => (
```

Comment:
^

Reply:
Post-discussed deemed not necessary.

Only used once. Iterates over Map.